client/src/main/java/se/fortnox/reactivewizard/client/HttpClientConfig.java

```diff
@@ -45,9 +45,11 @@
     private long maxRequestTime          = TimeUnit.MILLISECONDS.convert(1, MINUTES);
 
     private boolean isHttps;
-    private int     retryCount    = 3;
-    private int     retryDelayMs  = 1000;
-    private int     readTimeoutMs = 10000;
+    private int     retryCount     = 3;
+    private int     retryDelayMs   = 1000;
+    private int     readTimeoutMs  = 10000;
+    @JsonProperty("unsafeSecure")
```

Comment:
I get the idea of naming the config like the setting in rxnetty, but isn't it a bit akward? I imagine something like "validateCertificates" or something like that would be easier to understand if you're not familiar with rxnetty.

Reply:
unsafeSecure -> insecure, how does that sound? Taken from curl docs on the -k parameter

Reply:
Like this? It doesn't feel obvious that it has to do with certificate validation, a lot of things about http clients could be insecure ;)
Also, insecure is a negation of secure which makes it even less clear.

httpClient:
  insecure: false?

Reply:
Ah, I see. I will change to "validateCertificates" and use a default of true.